### PR TITLE
Finish armies.jsons

### PR DESCRIPTION
**Pull request by mpolakow**
Both armies json files are now finished.

### CODE REVIEW

File: src/resources/armies/moloch.json [hunk 1]
```diff
@@ -0,0 +1,332 @@
+{
```
Comment (tom3q) [inline src/resources/armies/moloch.json]: Hmm, we already added moloch before. Why is this commit adding it from scratch again?

Comment (tom3q) [inline src/resources/armies/moloch.json:1]: We already added moloch in a previous commit. Why is this one adding it again?

File: src/resources/armies/moloch.json [hunk 2]
```diff
@@ -0,0 +1,332 @@
...
+  "id": "moloch",
+  "name": "Moloch",
+  "tokens": [
+    {
+      "id": "hq",
+      "hq": true,
+      "initiative": 0,
+      "attacks": [
+        {
+          "type": "riffle",
```
Comment (tom3q) [inline src/resources/armies/moloch.json]: Does moloch HQ shoot? :P

File: src/resources/armies/moloch.json [hunk 3]
```diff
@@ -0,0 +1,332 @@
...
+          "angle": -1
+        }
+      ],
+      "modifiers": [
+        {
+          "type": "strengthr",
```
Comment (tom3q) [inline src/resources/armies/moloch.json:17]: Maybe just call it "riffle"?

File: src/resources/armies/moloch.json [hunk 4]
```diff
@@ -0,0 +1,332 @@
...
+          "self": true
+        }
+      ]
+    }, {
+      "id": "blocker",
+      "count": 2,
+      "initiative": 0,
+      "health": 3,
+      "shields": [
+       0 ]
```
Comment (tom3q) [inline src/resources/armies/moloch.json]: I guess we could just keep this in the line above.

File: src/resources/armies/moloch.json [hunk 5]
```diff
@@ -0,0 +1,332 @@
...
+      "health": 2,
+      "attacks": [
+        {
+          "type": "cannon",
+          "damage": 1
+        }
+      ],
+      "modifiers": [
+	{
+	  "type": "nostrength",
```
Comment (tom3q) [inline src/resources/armies/moloch.json]: What's this? :P

File: src/resources/armies/moloch.json [hunk 6]
```diff
@@ -0,0 +1,332 @@
...
+	  "self": true
+	}
+      ]
+    }, {
+      "id": "juggernaut",
+      "health": 2,
+      "count": 1,
```
Comment (tom3q) [inline src/resources/armies/moloch.json:50]: Can skip this for single tokens.

File: src/resources/armies/moloch.json [hunk 7]
```diff
@@ -0,0 +1,332 @@
...
+      "initiative": 1,
+      "attacks": [
+        {
+          "type": "melee",
+          "damage": 2
+        }, {
+	  "type": "riffle",
+	  "angle": 1,
+	  "damage": 1
```
Comment (tom3q) [inline src/resources/armies/moloch.json]: 1 is default so we can skip.

File: src/resources/armies/moloch.json [hunk 8]
```diff
@@ -0,0 +1,332 @@
...
+	}
+      ],
+      "shields": [
+	0, 2, 4 
+     ]
```
Comment (tom3q) [inline src/resources/armies/moloch.json:63]: Wrong indentation.

File: src/resources/armies/moloch.json [hunk 9]
```diff
@@ -0,0 +1,332 @@
...
+    }, {
+      "id": "the_clown",
+      "initiative": 2,
+      "health": 2,
+      "count": 1,
+      "attacks": [
+        {
+          "type": "melee",
+          "damage": 1
```
Comment (tom3q) [inline src/resources/armies/moloch.json:72]: Ditto.

File: src/resources/armies/moloch.json [hunk 10]
```diff
@@ -0,0 +1,332 @@
...
+        }, {
+          "type": "melee",
+          "angle": 5
+        }
+      ],
+      "modifiers": [
```
Comment (tom3q) [inline src/resources/armies/moloch.json]: Modifier is something that affects the neighboring tokens. I think we need a separate field for this kind of abilities. Perhaps "abilities"?

File: src/resources/armies/moloch.json [hunk 11]
```diff
@@ -0,0 +1,332 @@
...
+        {
+	  "type": "airstrike",
+	  "value": 1,
+	  "self": true
```
Comment (tom3q) [inline src/resources/armies/moloch.json]: These 2 values probably won't be needed if we move to "abilities".

File: src/resources/armies/moloch.json [hunk 12]
```diff
@@ -0,0 +1,332 @@
...
+      "initiative": 1,
+      "health": 2,
+      "count": 1,
+      "attacks": [
+        {
+          "type": "riffle",
+          "damage": 1
+        }, {
+	  "type": "riffle",
+	  "angel": 1
```
Comment (tom3q) [inline src/resources/armies/moloch.json]: angle

There are few others typos like this.

File: src/resources/armies/moloch.json [hunk 13]
```diff
@@ -0,0 +1,332 @@
...
+      "health": 2,
+      "attacks": [
+	{
+ 	   "type": "riffle",
+	   "damage": 1
+	}
+      ],
+      "modifiers": [
+	{
+	  "type": "second",
```
Comment (tom3q) [inline src/resources/armies/moloch.json:186]: We might want to have this in "abiities" instead.

File: src/resources/armies/moloch.json [hunk 14]
```diff
@@ -0,0 +1,332 @@
...
+     ]
+    }, {
+     "id": "the_brain",
+     "count": 1,
+     "health": 1,
+     "module": true,
+     "modifiers": [
+	{
+           "type": "strengthm",
+	   "value": 1
```
Comment (tom3q) [inline src/resources/armies/moloch.json:212]: We should make 1 the default, so we can skip for the majority of tokens.

File: src/resources/armies/moloch.json [hunk 15]
```diff
@@ -0,0 +1,332 @@
...
+          "type": "riffle",
+          "angle": -1
+        }
+      ],
+      "modifiers": [
+        {
+          "type": "strengthr",
+          "angle": -1,
+          "value": 1,
+          "self": true
```
Comment (tom3q) [inline src/resources/armies/moloch.json:20]: I guess we want to invert this by replacing "self" with "hostile", as most of the modifiers actually apply to allies only.

**Review by tom3q — CHANGES_REQUESTED** [review]

File: src/resources/armies/borgo.json [hunk 16]
```diff
@@ -28,6 +28,11 @@
         {
           "type": "riffle"
         }
+      ],
+      "modifiers": [
```
Comment (tom3q) [inline src/resources/armies/borgo.json]: abilities

File: src/resources/armies/borgo.json [hunk 17]
```diff
@@ -60,10 +65,151 @@
...
       ]
+    }, {
+      "id": "mutant",
+      "initiative": 2,
+      "health": 2,
+      "count": 6,
+      "attacks": [
+        {
+          "type": "melee",
+          "damage": 1
```
Comment (tom3q) [inline src/resources/armies/borgo.json]: 1 is the default.

File: src/resources/armies/borgo.json [hunk 18]
```diff
@@ -60,10 +65,151 @@
...
+	}, {
+	  "type": "strengthm",
+	  "angle": 1,
+	  "value": 1
+	}, {
+	  "type": "strengthm",
+	  "angle": 5,
+	  "value": 1
+	}, {
+	  "type": "toughness",
```
Comment (tom3q) [inline src/resources/armies/borgo.json]: I don't think super officer has such modifier.

File: src/resources/armies/borgo.json [hunk 19]
```diff
@@ -60,10 +65,151 @@
...
+      "attacks": [
+        {
+          "type": "melee",
+          "damage": 2
+        }
+      ]
+    }, {
+      "id": "super_officer",
+      "count": 1,
+      "health": 1,
```
Comment (tom3q) [inline src/resources/armies/borgo.json]: 2?

**Review by tom3q — CHANGES_REQUESTED** [review]
Also remove the explicit values that are the same as defaults.

File: src/resources/armies/borgo.json (hunk 19 above)
Comment (mpolakow) [reply src/resources/armies/borgo.json]: In 2.5 rules there only one of those

File: src/resources/armies/moloch.json (hunk 2 above)
Comment (mpolakow) [reply src/resources/armies/moloch.json]: Yes it does! (at least for now)

File: src/resources/armies/moloch.json (hunk 3 above)
Comment (mpolakow) [reply src/resources/armies/moloch.json:17]: TBD in future, left for now

File: src/resources/armies/moloch.json (hunk 4 above)
Comment (mpolakow) [reply src/resources/armies/moloch.json]: changed as requested

File: src/resources/armies/moloch.json (hunk 5 above)
Comment (mpolakow) [reply src/resources/armies/moloch.json]: As per conversation attack type would be enough to cover this modifier, removed

File: src/resources/armies/moloch.json (hunk 11 above)
Comment (mpolakow) [reply src/resources/armies/moloch.json]: Changed to abilities and removed extra values

File: src/resources/armies/moloch.json (hunk 12 above)
Comment (mpolakow) [reply src/resources/armies/moloch.json]: All should be fixed now

File: src/resources/armies/borgo.json (hunk 16 above)
Comment (mpolakow) [reply src/resources/armies/borgo.json]: changed

File: src/resources/armies/borgo.json (hunk 17 above)
Comment (mpolakow) [reply src/resources/armies/borgo.json]: Removed damage tag everywhere where it was 1

File: src/resources/armies/borgo.json (hunk 18 above)
Comment (mpolakow) [reply src/resources/armies/borgo.json]: It is described like this in player help sheet but its the same as extra health point, modifier removed.

File: src/resources/armies/borgo.json (hunk 19 above)
Comment (mpolakow) [reply src/resources/armies/borgo.json]: 2

**Comment by Nucleoprotein** [comment]
In team play moloch should also damage, net and push his partner units

**Comment by tom3q** [comment]
> In team play moloch should also damage, net and push his partner units

We should probably have some top level flag, like
`"hostile": true,`

I guess we'll figure out once the implementation gets to the point at which this starts to matter. :P

**Comment by tom3q** [comment]
Rebased manually. Please be more careful next time.

**Pull request closed without merging**